src/models/neural_networks/ealstm.py

```diff
@@ -297,8 +301,19 @@ def forward(
             static_x.append(static)
         if self.include_pred_month:
             static_x.append(pred_month)
+
+        # append prev_y to DYNAMIC data
         if self.include_prev_y:
-            static_x.append(prev_y)
+            # TODO: Gabi can you check this ?
```

Comment:
**IMPORTANT**: is this doing what I think it is re: appending the `prev_y` value to the dynamic data

Reply:
so this appends prev_y to each dynamic datastep.

Do we want to do this? I see why we would want static data to (sometimes) be globally static, but I also think keeping the previous behaviour (at least as an option) is desirable

Reply:
why would it be desirable though? Because that data is still being seen by the model?

Reply:
I do think that 'static' data should remain temporally static. 

The previous behaviour means that 'static' data is only static with regards to that particular target, however, it changes as the target changes, and hance analysing it requires you to analyse it over time.

Resolving this